wazo_sysconfd/plugins/asterisk/http.py

```diff
@@ -33,3 +33,8 @@ def move_voicemail(
             new_mailbox=new_mailbox,
         ),
     )
+
+
+@router.delete('/voicemails_context', status_code=200)
+def delete_voicemails_context(context: str, asterisk: Asterisk = Depends(get_asterisk)):
```

Comment:
Some validation should happen here. Maybe the same regex as what we use to validate context names in wazo-confd.
For example:
What happens if someone sends the request with the following body `{'context': '../../../../etc/passwd'}`

Reply:
Fixed